### ISSUE TITLE
Discussion Post: Contrasting Layer 2 Solutions

### ISSUE THREAD

**Issue by 29988122** [smartcontractresearchforum/research_content#198]
The Layer2 discussion happened around Auguest in the discord server's general channel.

Later I was asked by James to see it can be converted into a mini-discussion post. 

Judging by the two facts at 2021/09/20:
1. Arbitrum mainnet just go online.
2. Rollups are zkp heavy, one of my expertise field. zkEVMs, zk languages, proving systems, and zk further being used on scalibility solutions other than general rollups(recursive snarks, cleint-side browser(wasm) runnable proving systems, parallelized zk provings, low level optimizing cryptography primitives such as ultraplonk or fflonk, zk porters by matterslab.....), these are very foreign concepts to the outsiders, and having a dumbed down explanation on our forum would be very much helpful. 
We desprately need an ELI5 post: *a post that would rule them all*, and it should be relatively easy for *regular ethereum developers* to understand. 

After reading this post, the target is to make it easy for them to choose which L2 ecosystem they'd like to build upon.

The current form of the draft can be found here, using markdown editor like hackmd, as there are collapsible sections in the article that it feels very awkward if using google docs for previewing.
https://hackmd.io/2WwMgNJGSVWROYJUh2iYyg?both
Research stage's done in the previous month, now I just need to shove in as many keywords as possible, and to make some inaccurate but easily understandable explanation sentences.

*HEAVY* editorial process should be taken by either Deane or James before publication, in order to polish this article to make it really feel natural to both native speakers and ESL(english as second language) ppl.

**Comment by jamesmcgirk** [comment]
Really looking forward to reading this one! Is it ready for peer review yet? @29988122

**Comment by 29988122** [comment]
Am considering adding some details from this digest https://github.com/smartcontractresearchforum/research_content/issues/145 , which just out a few days ago.
But this is an introductory level post so...yeah the same conflict as always. Gotta decide the depth/length and strike a good balnce.

Will update you when I cleared all the clutter in the working doc.

**Comment by jamesmcgirk** [comment]
Discussed during our Monday meeting, @cipherix has taken a look and discussed some of the issues.

**Comment by encore9870** [comment]
Reopening when we settle with a new author to take over the draft
